tests/components/unifi/test_controller.py

```diff
@@ -147,31 +159,22 @@ async def test_controller_setup(hass):
         controller = await setup_unifi_integration(hass)
 
     entry = controller.config_entry
-    assert len(forward_entry_setup.mock_calls) == len(
-        unifi.controller.SUPPORTED_PLATFORMS
-    )
-    assert forward_entry_setup.mock_calls[0][1] == (entry, "device_tracker")
-    assert forward_entry_setup.mock_calls[1][1] == (entry, "sensor")
-    assert forward_entry_setup.mock_calls[2][1] == (entry, "switch")
+    assert len(forward_entry_setup.mock_calls) == len(SUPPORTED_PLATFORMS)
+    assert forward_entry_setup.mock_calls[0][1] == (entry, TRACKER_DOMAIN)
+    assert forward_entry_setup.mock_calls[1][1] == (entry, SENSOR_DOMAIN)
+    assert forward_entry_setup.mock_calls[2][1] == (entry, SWITCH_DOMAIN)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)